src/components/ProductDetail.js

```diff
@@ -0,0 +1,118 @@
+import React, {
+    Component, PropTypes,
+    StyleSheet,
+    PixelRatio,
+    ToolbarAndroid, TouchableOpacity,
+    ScrollView, View,
+    Text, Image,
+} from 'react-native';
+
+const styles = StyleSheet.create({
+    contentContainer: {
+        padding: 10,
+    },
+    toolbar: {
+        backgroundColor: '#a9a9a9',
+        height: 56,
+    },
+    mainSection: {
+        flexDirection: 'row',
+    },
+    detailsImage: {
+        width: 134,
+        height: 200,
+        backgroundColor: '#eaeaea',
+        marginRight: 10,
+    },
+    rightPane: {
+        justifyContent: 'space-between',
+        flex: 1,
+    },
+    productReference: {
+        flex: 1,
+        fontSize: 16,
+        fontWeight: '500',
+    },
+    small: {
+        fontSize: 12,
+    },
+    pane: {
+        marginTop: 10,
+    },
+    paneTitle: {
+        fontSize: 14,
+    },
+    paneStrongValue: {
+        fontSize: 28,
+        fontWeight: '500',
+        color: 'green',
+    },
+    separator: {
+        backgroundColor: 'rgba(0, 0, 0, 0.1)',
+        height: 1 / PixelRatio.get(),
+        marginVertical: 10,
+    },
+});
+
+class ProductDetail extends Component {
```

Comment:
I have the following error when I make this component stateless, so I revert it at https://github.com/marmelab/Posters_Galore_Android/commit/247bbd3e5a53400b2a20ae692028f0e362a7030c

![Error](http://image.noelshack.com/fichiers/2015/47/1447947948-stateless-error.png)